api-server/package.json

```diff
@@ -91,9 +91,6 @@
     "babel-plugin-transform-imports": "^1.5.1",
     "commitizen": "^2.9.6",
     "cz-freecodecamp": "^1.0.1",
-    "eslint": "^4.10.0",
-    "eslint-config-freecodecamp": "^1.1.1",
```

Comment:
do we not follow the freecodecamp rules anymore?

Reply:
I think the intent is to add them to the root of the repo. I am all for moving all dev deps to the root.

Reply:
But linting was done without these rules.

Reply:
Ok, I was wrong and it should be using config from the root of the project, but I don't understand why did it insert trailing comma here https://github.com/freeCodeCamp/freeCodeCamp/pull/35076/files#diff-d4492df91628c2d780b70df6ad7944aaR21

Reply:
That may have been my editor no picking up the `.prettierrc`.

Reply:
I have swapped this out for an updated `@freecodecamp/eslint-config` package which brings the config up to date with the latest eslint/babel-eslint/plugins.